app/src/components/blocks/documentInputForm/DocumentInputForm.tsx

```diff
@@ -75,12 +75,23 @@ const DocumentInputForm = ({
                                 .
                             </li>
                         )}
+                        {formType === DOCUMENT_TYPE.LLOYD_GEORGE && (
+                            <>
+                                <li>
+                                    Each Lloyd George file uploaded must match the following format:
+                                    [PDFnumber]_Lloyd_George_Record_[Patient Name]_[NHS
+                                    Number]_[D.O.B]. For example: 1of2_Lloyd_George_Record_[Joe
+                                    Bloggs]_[123456789]_[25-12-2019]
```

Comment:
Are we keeping the brackets in the actual name of the file? The example suggests otherwise as we've not got the PDFNumber in brackets but everything else is?

Reply:
keeping it for the moment